README.md

```diff
@@ -49,15 +49,15 @@ These are described below.  We recommend installing in a managed Python environm
     1. Activate your environment.
     1. `pip install git+https://github.com/brain-score/brainio.git`
 1. Install a project that provides a BrainIO Catalog
-    * If you intend to use a BrainIO catalog to access data or stimuli that have already been packaged, you will need to install the project that provides the catalog that you intend to use.  Here we illustrate with a project called `brainio-dicarlo` provided by the DiCarlo Lab:  
+    * If you intend to use a BrainIO catalog to access data or stimuli that have already been packaged, you will need to install the project that provides the catalog that you intend to use.  Here we illustrate with [a project called `brainio-dicarlo`](https://github.com/dicarlolab/brainio-dicarlo) provided by the DiCarlo Lab:  
     1. Activate your environment
     1. `pip install git+https://github.com/dicarlolab/brainio-dicarlo.git`
     * This registers the catalog with the Python interpreter in your environment 
 so that the BrainIO tools can use it.  
 
 ## Install BrainIO As A Provider Of Packages
 
-If you intend to package Stimulus Sets And Data Assemblies and add them to a project's catalog, instead of installing the project as above, you will need to clone the repository for that project and install it in your environment in editable mode.  In the example below we set up to add to `brianio-dicarlo`'s BrainIO Catalog:  
+If you intend to package Stimulus Sets And Data Assemblies and add them to a project's catalog, instead of installing the project as above, you will need to clone the repository for that project and install it in your environment in editable mode.  In the example below we set up to add to the BrainIO Catalog provided by [a project called `brianio-dicarlo`](https://github.com/dicarlolab/brainio-dicarlo):  
```

Comment:
I still don't know what brianio is

Reply:
He's just this guy I know.  Figured I'd give him a shoutout, but fine, have it your way.  😆